source-hubspot-native/source_hubspot_native/api.py

```diff
@@ -271,14 +460,260 @@ async def fetch_recent_tickets(
     log: Logger, http: HTTPSession, since: datetime, cursor: PageCursor
 ) -> tuple[Iterable[tuple[datetime, str]], PageCursor]:
 
-    url = f"{HUB}/crm-objects/v1/change-log/tickets"
-    params = {"timestamp": int(since.timestamp() * 1000) - 1}
+    url = f"{HUB}/crm/v3/objects/tickets"
+    
+    result = OldRecentTicket.model_validate_json(
+        await http.request(log, url, params=None)
+    )
+
+    return (
+        (r.updatedAt, str(r.id))
+        for r in result.results
+    ), None and None
+
+async def fetch_recent_contacts_lists(
+    log: Logger, http: HTTPSession, since: datetime, page: PageCursor
+) -> tuple[Iterable[tuple[datetime, str, Any]], PageCursor]:
+
+    url = f"{HUB}/contacts/v1/lists"
+    params = {"count": 100, "offset": page} if page else {"count": 250}
```

Comment:
We're using `{"count": 1}` elsewhere if there is no pre-existing page cursor. Why 250 here?

Reply:
Hubspot limits the amount of lists by 250 ( if you have Pro/Enterprise it seems to handle more), and data from this endpoint is really small. If no page is given, instead of creating a pagination object with 100 data and have it pass 3 times, i just saw more simpler to handle all 250 lists at once.

If there are more lists, pagination will occur normally